src/mem_table_flush.rs

```diff
@@ -108,7 +108,7 @@ impl MemtableFlusher {
             let id = SsTableId::Compacted(Ulid::new());
             let sst_handle = self
                 .db_inner
-                .flush_imm_table(&id, imm_memtable.table())
+                .flush_imm_table(&id, imm_memtable.table(), true)
```

Comment:
I went back-and-forth on making this a configurable property. It seems like it should be a benefit for most use-cases, but I could imagine some outliers where we may not want to enable this.

Reply:
+1 on hard coding for now. CLEAN_SLATE.md:

> We will hard-code some configurations to keep the number of user-facing options low.